### ISSUE TITLE
Special Characters in accounts.cfg cause restart-loop

### ISSUE THREAD

**Issue by danielandross** [pyload/pyload#4517]
### Description
Adding a hoster account in settings will write the password to accounts.cfg. If that contains special characters. Next start will cause a boot loop. 
When accounts.cfg is deleted pyload starts again properly.
### Debug log
There is no error message indicating an error or stack trace. You see that the startup repeats indefinitely.

### Additional references

Could you support escaping characters?

**Comment by GammaC0de** [comment]
> If that contains special characters

For characters exactly?

**Comment by danielandross** [comment]
I was wrong about the special characters. I did another test:
1. deleting accounts.cfg file to have it recreated by application
2. starting pyload - it works fine
3. In app I added a filer.net account with user "user" and password "pass" (no special characters included) app keeps running.
4. at next start of the app will loop

[bad_accounts.zip](https://github.com/user-attachments/files/17254517/bad_accounts.zip)
